lib/atlas/preset/scaling.rb

```diff
@@ -17,6 +17,10 @@ class Scaling
       validates :area_attribute, presence: true
       validates :value,          presence: true, numericality: true
       validates :base_value,     presence: true, numericality: true
+
+      def factor
+        value.to_r / base_value.to_r
```

Comment:
Right you are; sorry. I must have ended up looking at the master branch when I wrote that. :blush: